darts-api-stub/cases/case.js

```diff
@@ -11,7 +11,37 @@ const singleCase = {
   judges: ['Judge Judy'],
   prosecutors: ['Polly Prosecutor'],
   defenders: ['Derek Defender'],
-  reporting_restriction: ['Section 4(2) of the Contempt of Court Act 1981'],
+  reporting_restriction: 'Section 4(2) of the Contempt of Court Act 1981',
+  reporting_restrictions: [
+    {
+      hearing_id: 1,
+      event_id: 123,
+      event_name: 'Section 4(2) of the Contempt of Court Act 1981',
+      event_text: '',
+      event_ts: '2023-08-21T09:00:00Z',
+    },
+    {
+      hearing_id: 1,
+      event_id: 123,
+      event_name: 'Section 39, Children and Young People Act 1933',
+      event_text: '',
+      event_ts: '2023-08-21T09:00:00Z',
+    },
+    {
+      hearing_id: 1,
+      event_id: 123,
+      event_name: 'Section 40, Children and Young People Act 1933',
+      event_text: '',
+      event_ts: '2023-08-21T09:00:00Z',
+    },
+    {
+      hearing_id: 1,
+      event_id: 123,
+      event_name: 'Restrictions lifted',
+      event_text: '',
+      event_ts: '2023-08-21T09:00:00Z',
+    },
+  ],
```

Comment:
These should have different `event_id`'s and ideally be over multiple hearing's with different dates.

Reply:
I'll fix this in the next PR if that's okay